tests/common/test_transform.py

```diff
@@ -5,6 +5,18 @@
 import tempfile
 
 
+def _define_binop(op, width):
+    return magma.Bits[width]._declare_binary_op(op)
```

Comment:
We may want to add an external API for accessing these (I think the problem is they're only available through the operators right now?)

Reply:
Possibly...this code is "bad"...we should just be using the operators here anyway. This is a result of using old gemstone abstractions which should go away.

That being said, we may want a public API for other clients/reasons.